index.bs

```diff
@@ -587,6 +587,18 @@ A <dfn>source type</dfn> is one of the following:
 
 </dl>
 
+<h3 id="event-level-report-window-header">Event-level report window</h3>
+
+An <dfn>event-level report window</dfn> is a [=struct=] with the following items:
+
+<dl dfn-for="event-level report window">
+: <dfn>start</dfn>
+:: A [=moment=].
+: <dfn>end</dfn>
+:: A [=moment=].
```

Comment:
Should we enforce that `end` is strictly greater than `start`?

Reply:
Sure, done